test/spectests/spectests.cpp

```diff
@@ -24,32 +24,36 @@ constexpr auto JsonExtension = ".json";
 constexpr auto UnnamedModule = "_unnamed";
 constexpr unsigned TestMemoryPagesLimit = (4 * 1024 * 1024 * 1024ULL) / fizzy::PageSize;  // 4 Gb
 
-// spectest module details:
-// https://github.com/WebAssembly/spec/issues/1111
-// https://github.com/WebAssembly/spec/blob/e3a3c12f51221daaa901691e674d7ff990029122/interpreter/host/spectest.ml#L33
-// https://github.com/WebAssembly/spec/blob/f85f6e6b70fff9ee0f6ba21ebcdad2183bd67874/test/harness/sync_index.js#L79
+// spectest module definition:
+// https://github.com/WebAssembly/spec/blob/master/interpreter/README.md#spectest-host-module
 /* wat2wasm
 (module
+  (global (export "global_i32") i32 (i32.const 666))
```

Comment:
Do you think this new ordering is dependent on? And if so do you think wabt produced the correct expected output?

If not, is this just to show it is a copy from a spec?

Reply:
It's just to show correspondence to the spec. Ordering of sections in the module is strict, all members are referred by names only in spectests, and it's not even required for module to be Wasm.

Reply:
> Ordering of sections in the module is strict

The sections are strictly ordered, but the wat->wasm translator could decide how it lays out members within those, e.g. the ordering of functions.

Reply:
Well many out unit tests assume that wabt puts them in the order as defined in wat.